config/initializers/active_job_logger.rb

```diff
@@ -0,0 +1 @@
+ActiveJob::Base.logger = nil if Rails.env.production?
```

Comment:
Is there some way we could suppress the PII without switching off `ActiveJob` logging altogether?

(unrelated: Would `config.active_job.logger` in `production.rb` be more idiomatic?)

Reply:
Not really, because of the way ActionMailer jobs pass arguments to the mailer (see Tijmen's comment/reply). If we switch to Sidekiq workers for mailing we could bring ActiveJob logging back, but then again, we won't be using ActiveJob.